src/key_packages/mod.rs

```diff
@@ -236,9 +236,16 @@ impl KeyPackageBundle {
         ciphersuite_name: CiphersuiteName,
         credential_bundle: &CredentialBundle,
         extensions: Vec<Box<dyn Extension>>,
+        ciphersuites: Option<&[CiphersuiteName]>,
```

Comment:
I think the documentation above needs to be adjusted to say that one can now control the capabilities extension via `ciphersuites`.